scripts/build_report.py

```diff
@@ -48,7 +49,8 @@ def transform_df(df, param_cols, tflops_col, hbm_col, benchmark, compiler, tag):
         n: os.getenv(n.upper(), default="")
         for n in ["libigc1_version", "level_zero_version", "gpu_device", "agama_version"]
     }
-    assert host_info['gpu_device'], "Could not find GPU device description, was capture_device.sh called?"
+    if not host_info['gpu_device']:
```

Comment:
Let's use consistent quotes within the file
```suggestion
    if not host_info["gpu_device"]:
```

Reply:
Done